.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
```

Comment:
TIP: We typically add all ``.vscode`` files as templates (i.e. ``launch.json.template``) and let the user decide. Notice that these configuration files might change from user to user and we do not want them under version control. For that reason, I would add this in ``.gitignore``
```.gitignore
# IDEs config (except templates)
.vscode/*
!.vscode/*.template.json
```